go/vrf/conversion_test.go

```diff
@@ -58,8 +58,8 @@ func TestSEG1EncodeDecode(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	b := secg1EncodeCompressed(c, Ax, Ay)
-	Bx, By, err := secg1Decode(c, b)
+	b := marshalCompressed(c, Ax, Ay)
```

Comment:
For the marshalling, unmarshalling test it would be nice to also include some standard values here. I think Wycheproof probably has coverage for this.

Reply:
Done

Reply:
Pulled from https://github.com/google/wycheproof/blob/master/testvectors/ecdh_secp256r1_test.json